services/audio-broadcaster/templates/ingress.yaml

```diff
@@ -0,0 +1,29 @@
+---
+{{- if eq .Values.serviceType "ClusterIP" }}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ .Release.Name }}-ingress
+  namespace: {{ .Values.namespace }}
+  {{- with .Values.ingress.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  {{- if .Values.ingress.className }}
+  ingressClassName: {{ .Values.ingress.className }}
+  {{- end }}
+  rules:
+    - host: {{ required "ingress.host must be set" .Values.ingress.hostname | quote }}
+      http:
+        paths:
+        {{- range $index, $mic := .Values.microphones }}
+          - path: {{ .Values.ingress.httpPath }}/{{ $index }}
+            pathType: {{ default "Prefix" .Values.ingress.pathType }}
+            backend:
+              service:
+                name: {{ include "audio-broadcaster.fullname" . }}-service-{{ $index }}
+                port:
+                  number: {{ .Values.ports.container }}
+        {{- end }}
```

Comment:
Is this correctly indented :thinking:?

Reply:
Yes, the indent looks right. You can always use `helm template` to check things

Reply:
Thanks for the new knowledge :pray: